Week 1 Recipe Page/styles.css

```diff
@@ -190,7 +216,7 @@ input[name="favorite_soup"]::placeholder {
     left: 0;
     color: #A17D58; 
     font-size: 1.5em; 
-    top: -4px; 
+    top: -5px; 
```

Comment:
Using alignment using top bottom can be tricky especially if the view is suppose to be mobile responsive.